compiler/luci/pass/include/luci/Pass/QuantizeWithMinMaxPass.h

```diff
@@ -30,8 +30,16 @@ namespace luci
 class QuantizeWithMinMaxPass : public logo::Pass
 {
 public:
-  QuantizeWithMinMaxPass(loco::DataType input_dtype, loco::DataType output_dtype)
-      : _input_dtype{input_dtype}, _output_dtype{output_dtype}
+  enum QuantizationGranularity
```

Comment:
Will this be different from `QuantizeDequantizeWeightsPass` in future?
If not how about put as individual enum ? maybe same level of `enum AlgorithmParameters` ?

Reply:
> Will this be different from QuantizeDequantizeWeightsPass in future?

I don't think so.

> If not how about put as individual enum ?

It would be possible to make a separate header file (e.g., QuantizationParameters.h) and make both Passes include it. But I think the current way is simpler, because AFAIK, no more parameters will be added for quantization.

> maybe same level of enum AlgorithmParameters ?

`Option`, where `AlgorithmParameters` is implemented, is used to parse arguments. So I think it is more proper to place `QuantizeGranularity` in the quantization passes (or a new file as mentioned above).

Reply:
Reason is it's a bit long to use like `str_to_granularity<luci::QuantizeWithMinMaxPass::QuantizationGranularity>(granularity))`
It would be better just to use `str_to_granularity(granularity)`

Reply:
I've simplified the interface. PTAL